trailblazer/services/analysis_service/analysis_service.py

```diff
@@ -73,6 +74,9 @@ def get_summaries(self, request_data: SummariesRequest) -> SummariesResponse:
         summaries: list[Summary] = []
         for order_id in request_data.order_ids:
             analyses: list[Analysis] = self.store.get_latest_analyses_for_order(order_id)
-            summary: Summary = create_summary(analyses=analyses, order_id=order_id)
+            analyses_without_rsync: list[Analysis] = [
+                analysis for analysis in analyses if analysis.workflow != Workflow.RSYNC
```

Comment:
This filtering might be done faster in the database query instead, but lets check if this adds a noticeable delay first

Reply:
but I think a filter for "not rsync" would become obsolete in the near future.

Reply:
Yeah, true. When we retrieve orders from cg, we do call this endpoint once per order which could motivate speeding it up a bit. But I agree, lets leave it

Reply:
Resolved in private chat to leave as is for now.